themes/linear/entries.typ

```diff
@@ -59,16 +60,27 @@
       inset: 10pt,
       [#context.title],
     )
-  ], footer: [
-    #set align(left)
-    Written by:
-    #h(30%)
-    Witnessed by:
-  ], background: set-margins(8%, 8%))
+  ], 
+  footer: [
+    #grid(
+      columns: (2fr, 2fr, 1fr),
+      [
+        Written by:

```

Comment:
Make sure to display the witness and author here if the user specifies those.

Reply:
Okay got it.